src/main/java/steps/TestResultsSteps.java

```diff
@@ -580,6 +580,13 @@ public void validateTestFieldExists(String fieldName) {
         assertThat(response.then().body("[0].testTypes[0]", hasKey(fieldName)));
     }
 
+    @Step
+    public void validateTestFieldNotPresent(String fieldName) {
+//        response.then().log().all();
```

Comment:
remove commented out line

Reply:
we'll keep these commented to easily uncomment for quick debugging purposes